recipes/conda-bash-completion/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "conda-bash-completion" %}
+{% set version = "1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: "{{ version }}"
+  url: https://github.com/tartansandal/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 82fa38dae6cec42095b5a1f82d5db41b35a7a2c35642a9b7ad548385ae601659
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - bash-completion
+  run:
+    - bash-completion
+
+test:
+    source_files:
+      - conda
+    commands:
+      # Ensure the library loads without error
+      - bash -e conda
+
+about:
+  home: https://github.com/tartansandal/conda-bash-completion
+  license: GPL3
+  license_file: LICENSE
+  summary: "Bash completion for the conda commmand."
+
+extra:
+  recipe-maintainers:
+    - tartansandal
+
+# vim:set ts=8 sw=2 sts=2 tw=78 et:
```

Comment:
Please remove these extra lines.

Reply:
I've done that, but now the linter is complaining about a missing `recipe-maintainers` section

Reply:
Right I just mean the vim comments.

Reply:
All good. Thanks for your help this :smile: